x/evidence/spec/06_begin_block.md

```diff
@@ -0,0 +1,100 @@
+<!--
+order: 6
+-->
+
+# BeginBlock
+
+## Evidence Handling
+
+Tendermint blocks can include
+[Evidence](https://github.com/tendermint/tendermint/blob/master/docs/spec/blockchain/blockchain.md#evidence),
+which indicates that a validator committed malicious behavior. The relevant information is
+forwarded to the application as ABCI Evidence in `abci.RequestBeginBlock` so that
+the validator an be accordingly punished.
+
+### Equivocation
+
+Currently, the evidence module only handles evidence of type `Equivocation` which is derived from
```

Comment:
@amaurymartiny I think we should add some mention of the new light client evidence handling here as well. cc @robert-zaremba @marbar3778 who may have more info on how that should look.

Reply:
yes this need updating